### ISSUE TITLE
FHIR may have IRIs for UMLS terms

### ISSUE THREAD

**Issue by graybeal** [ncbo/bioportal-project#144]
A Slack communication with Jerven Bolleman of SIB / SwissProt:

[My use of BioPortal is] Finding the correct IRIs of important shared vocabularies
Negative, is non official ontologies being above or missleading
e.g. http://purl.bioontology.org/ontology/SNOMEDCT/123037004
Instead of http://snomed.info/id/123037004

browser.ihtsdotools.orgbrowser.ihtsdotools.org
SNOMED International Browser
SNOMED International SNOMED CT Browser

John Graybeal  12:06 PM
ewww. what's the pattern of that—lots in SNOMED, or in many UMLS ontologies, or just scattered occurrences?

Jerven Bolleman  12:07 PM
Basically, with FHIR. Loinc etc. have official supported IRI patterns
With supported infra
Maybe not a bioportal issue (edited)

**Comment by graybeal** [comment]
Wonder if these could be pushed back into UMLS?  (And what would the implications be for BioPortal, if the identifiers for all those terms changed? Might be unfortunate.)

**Comment by alexskr** [comment]
also see to https://github.com/ncbo/umls2rdf/issues/43
